server/opts.go

```diff
@@ -619,6 +622,10 @@ e.g.
             "CurveP384",
             "CurveP521"
         ]
+
```

Comment:
Should make conditional for only windows built servers.

Reply:
Will augment the usage only on windows platform.

Reply:
On windows only, `nats-server --help_tls true` will include the following usage addendum:

```
In place of cert_file and key_file you may use the windows certificate store:

    tls {
        cert_store:     "WindowsCurrentUser"
        cert_match_by:  "Subject"
        cert_match:     "MyServer123"
    }
```